src/Init/Init_ByFile.cpp

```diff
@@ -43,10 +43,17 @@ extern void (*Flu_ResetByUser_API_Ptr)( const int lv, const int FluSg, const int
 //                               (e.g., entropy) directly instead of loading it from the disk
 //                               --> Only NCOMP_TOTAL-1 (which equals 5+NCOMP_PASSIVE_USER currently) fields
 //                                   should be stored in "UM_IC"
-//                           (2) For ELBDM, we will calculate the density field from the input wave function
-//                               directly instead of loading it from the disk
+//                           (2) For ELBDM:
 //                               --> Only NCOMP_TOTAL-1 (which equals 2+NCOMP_PASSIVE_USER currently) fields
 //                                   should be stored in "UM_IC"
+//
+//                                  ELBDM_SCHEME == ELBDM_WAVE:
+//                                     We will load the real and imaginary parts from the disk on all levels
+//                                     and calculate the density field from the input wave function
+//                                     directly instead of loading it from the disk.
+//                                  ELBDM_SCHEME == ELBDM_HYBRID
+//                                     We will load the density and phase fields from the disk on all levels
+//                                     There is no need to separately calculate the density field.
```

Comment:
@hyschive 
Yes, it does not work correctly. I have not thought about this function. When is it actually used? What would be the expected data format for the hybrid scheme? The same as for the LSS test problem? imagine it will not work for `OPT__UM_IC_LEVEL >= ELBDM_FIRST_WAVE_LEVEL`

Reply:
@KunkelAlexander
- `Init_ByFile_Default()` is used when `Init_ByFile_User_Ptr` is not reset by a test problem initializer (cf. [example](https://github.com/KunkelAlexander/my-gamer-fork/blob/spectral_refinement/src/TestProblem/ELBDM/LSS/Init_TestProb_ELBDM_LSS.cpp#L267) in `LSS`). I think we can simply adopt the same data format as the LSS test problem.
- We first load data at lv `OPT__UM_IC_LEVEL` [here](https://github.com/KunkelAlexander/my-gamer-fork/blob/spectral_refinement/src/Init/Init_ByFile.cpp#L281) instead of lv 0. Then, if `OPT__UM_IC_NLEVEL>1`, we load data from lv `OPT__UM_IC_LEVEL+1` to `OPT__UM_IC_LEVEL+OPT__UM_IC_NLEVEL-1` [here](https://github.com/KunkelAlexander/my-gamer-fork/blob/spectral_refinement/src/Init/Init_ByFile.cpp#L300). After that, we set data on lv 0 to `OPT__UM_IC_LEVEL-1` by restriction [here](https://github.com/KunkelAlexander/my-gamer-fork/blob/spectral_refinement/src/Init/Init_ByFile.cpp#L353). So the key is that levels below `OPT__UM_IC_LEVEL` are set by _restriction_ instead of `UM_IC`.

   If you agree that the current implementation does not support `OPT__UM_IC_LEVEL >= ELBDM_FIRST_WAVE_LEVEL` when enabling `ELBDM_MATCH_PHASE`, we can simply raise an error.

Reply:
@hyschive
Thank you for the explanation. I agree that the current implementation will not work in this case. I believe I have fixed the problem with `Init_ByFile_Default()` that you pointed out. 
In addition, `Aux_Check_Parameter` now raises the following error
```
 if ( OPT__INIT == INIT_BY_FILE && OPT__UM_IC_LEVEL >= ELBDM_FIRST_WAVE_LEVEL && ELBDM_MATCH_PHASE )
      Aux_Error( ERROR_INFO, "ELBDM_HYBRID currently does not support loading both fluid and wave levels from UM_IC because of phase matching !!\n" );
```